### ISSUE TITLE
in2csv complains about missing monkeypatch

### ISSUE THREAD

**Issue by fgregg** [wireservice/csvkit#686]
csvkit==1.0.0

```python
> in2csv "College Data - Uptake.xlsx" > college_b.csv
/usr/lib/python3/dist-packages/apport/report.py:13: PendingDeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
Traceback (most recent call last):
  File "/usr/local/bin/in2csv", line 9, in <module>
    load_entry_point('csvkit==1.0.0', 'console_scripts', 'in2csv')()
  File "/usr/local/lib/python3.5/dist-packages/pkg_resources/__init__.py", line 564, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/local/lib/python3.5/dist-packages/pkg_resources/__init__.py", line 2608, in load_entry_point
    return ep.load()
  File "/usr/local/lib/python3.5/dist-packages/pkg_resources/__init__.py", line 2268, in load
    return self.resolve()
  File "/usr/local/lib/python3.5/dist-packages/pkg_resources/__init__.py", line 2274, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/local/lib/python3.5/dist-packages/csvkit/utilities/in2csv.py", line 12, in <module>
    agatedbf.patch()
  File "/usr/local/lib/python3.5/dist-packages/agatedbf/__init__.py", line 12, in patch
    agate.Table.monkeypatch(TableDBF)
AttributeError: type object 'Table' has no attribute 'monkeypatch'
```

**Comment by jpmckinney** [comment]
Can you create a fresh virtualenv and run `pip freeze`?

**Comment by fgregg** [comment]
```bash
agate==1.5.0
agate-dbf==0.1.0
agate-excel==0.1.0
awesome-slugify==1.6.5
Babel==2.3.4
csvkit==1.0.0
dbfread==2.0.7
et-xmlfile==1.0.1
isodate==0.5.4
jdcal==1.3
leather==0.3.2
lxml==3.6.4
openpyxl==2.4.1
parsedatetime==2.1
pkg-resources==0.0.0
python-dateutil==2.2
pytimeparse==1.1.5
pytz==2016.7
regex==2016.11.21
six==1.10.0
SQLAlchemy==1.1.4
Unidecode==0.4.19
xlrd==1.0.0
```

**Comment by jpmckinney** [comment]
Aha. `agate` can't be 1.5.0. Needs to be 1.4.0 until `agate-dbf` is updated.

**Comment by jpmckinney** [comment]
If you create a patch to limit `agate` to <1.4 I'll merge.

**Comment by jpmckinney** [comment]
Fixed by eb8d113da3aefb0fac9dbdab1663160d15e5488f

**Comment by fgregg** [comment]
Thanks!
